.github/renovate.json5

```diff
@@ -36,6 +56,14 @@
       matchStrings: [
         "# renovate: datasource=(?<datasource>\\S+)\n\\s+(?<depName>.*?)==(?<currentValue>.*?)\\n"
       ]
+    },
+    {
+      // Make everything in requirements-dev.txt a dev dependency.
```

Comment:
I'm not sure I follow: can you explain what it means to be a "dev dependency" in the context of this configuration?

Reply:
In this context it means this file itself will be updated with development dependencies, not with regular dependencies. This was the best compromise I could think of to still group updates to things like black/pylint/flake8/etc. without creating a list of packages that are development dependencies somewhere.

In practice, this file will be updated primarily in PRs made through the rules on lines 31-39 and 45-50. By de-prioritising dev dependency PRs, I *believe* I've made it so higher priority PRs can scoop up updates in this file (e.g. updating requests in `requirements.txt` should still update it here too), but I'm not entirely sure on that.

The intention, though, is that minor updates to dev tools get auto-merged, and likewise minor updates to regular dependencies get auto-merged into `requirements-dev` but not `requirements` (though once we're through with the initial barrage of updates these PRs should be created at the same time), so we'll be using the newer minor updates on our own machines before they get included in the standard package.

I hope that makes sense. I feel like some of this probably needs to get added to the comments here, but I'm not sure what's useful and what's just yammering, so I'd love some help turning this braindump into a usable explanation.